src/pages/steps/AdditionalInformationStep/PriceInformation.tsx

```diff
@@ -267,14 +267,16 @@ const PriceInformation = ({
                 >
                   <Inline
                     width="100%"
-                    alignItems="baseline"
                     paddingY={3}
+                    alignItems="baseline"
                     justifyContent="flex-start"
                     spacing={3}
                   >
-                    <Inline minWidth="30%">
+                    <Inline minWidth="60%">
                       {rate.category === PriceCategory.BASE && (
-                        <Text width="100%">{rate.name[i18n.language]}</Text>
+                        <Text width="100%">
+                          Kansentarief met UiTPAS Van Polder tot Kempen
```

Comment:
This probably needs to be translated?

Reply:
Ooh crap, yes I used this string since it was a reference for deciding on the width